ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -133,37 +133,24 @@ const syncAndReloadTraining = fp.flow(
 });
 
 // Helper Functions
-// The module has already expired
-export const hasExpired = (expiration: number): boolean =>
-  !!expiration && getWholeDaysFromNow(expiration) < 0;
-
-// The module can either be expired or is expiring
-export const isExpiring = (expiration: number): boolean =>
-  expiration
-    ? getWholeDaysFromNow(expiration) <=
-      serverConfigStore.get().config.accessRenewalLookback
-    : false;
-
-const isModuleExpiring = (status: AccessModuleStatus): boolean =>
-  isExpiring(status?.expirationEpochMillis);
-
-export const isExpiringNotBypassed = (moduleStatus: AccessModuleStatus) => {
-  return isModuleExpiring(moduleStatus) && !moduleStatus.bypassEpochMillis;
-};
 
 const isExpiringAndNotBypassed = (
```

Comment:
should the method name also reflect isExpiringOrExpired? but would become a really loong name 
isExpiringOrExpiredAndNotBypassed